### PR TITLE
Create outreach-campaign-request

### PR DESCRIPTION
**Pull request by o-ram**
Draft of template for outreach campaign issue.

### CODE REVIEW

**Comment by o-ram** [comment]
@AnthonyRollins I believe I created a new issue template for an outreach campaign request. Could you please review?

**Review by AnthonyRollins — CHANGES_REQUESTED** [review]
Taking a look at this file - it doesn't appear to be working, and I think it is because the file needs to be a YAML file.

To fix this: add `.yml' to the end of the file name. Let me know if you need some help with this.

**Comment by o-ram** [comment]
@AnthonyRollins ok, I think I successfully added .yml to the file name, but LMK if I did not do it right

File: .github/ISSUE_TEMPLATE/outreach-campaign-request.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      required: true
+  - type: textarea
+    id: goal
+    attributes:
+      label: What is the goal of this campaign?
+      description: Input as much detail as possible.
+    validations:
+      required: true
+  - type: textarea
+    id: goal
```
Comment (AnthonyRollins) [inline .github/ISSUE_TEMPLATE/outreach-campaign-request.yml:51]: Your ids need to be unique so this one needs to change

**Review by AnthonyRollins — CHANGES_REQUESTED** [review]
Ids need to be unquie

File: .github/ISSUE_TEMPLATE/outreach-campaign-request.yml (hunk 1 above)
Comment (o-ram) [reply .github/ISSUE_TEMPLATE/outreach-campaign-request.yml]: I changed one of them!

**Pull request merged**